### ISSUE TITLE
Redirecionar o @@search para o @@busca

### ISSUE THREAD

**Issue by idgserpro** [plonegovbr/brasil.gov.portal#278]
O brasil.gov.portal customiza o formulário de busca para utilizar o [@@busca](https://github.com/plonegovbr/brasil.gov.portal/blob/master/src/brasil/gov/portal/browser/viewlets/templates/searchbox.pt#L9) ao invés de utilizar o @@search. Para ficar coerente, precisamos redirecionar as outras requisições ao @@search para o @@busca. Um exemplo de onde o @@search é utilizado, é na [busca de tags](https://github.com/plone/plone.app.layout/blob/master/plone/app/layout/viewlets/keywords.pt#L16).

Precisamos customizar o @@search e redirecionar para o @@busca.

**Comment by hvelarde** [comment]
acho que isso pode ser resolvido mais facilmente no plone.app.redirector; ver #327.

**Comment by hvelarde** [comment]
sinceramente agora acho melhor remover completamente a customização.

**Comment by agnogueira** [comment]
Acho que é uma correção impotante. O @@search é um página que não foi trabalhada nem formatada. Todos os ajustes de busca estão no @@busca

**Comment by hvelarde** [comment]
sim, mas essa escolha está provocando um monte de customizações que poderiam ser evitadas; não sei até qual ponto é válido isso hoje e não usar `role="search"`, por exemplo.

isso tem que ser reavaliado, para mim não faz sentido mas não sou um experto no tema.

**Comment by agnogueira** [comment]
Foi uma recomendação de acessibilidade na época.  Também não sei quem pode bater o martelo sobre isso, mas acho que temos um bug aqui e não melhoria, dado que a @@search é uma página visualmente quebrada.
